examples/kv-server/src/main/java/com/yahoo/gondola/demo/DemoApplication.java

```diff
@@ -62,6 +65,17 @@ protected void configure() {
         // register resources in the package
         packages(true, "com.yahoo.gondola.demo");
         instance = this;
+
+        Runtime.getRuntime().addShutdownHook(new Thread() {
+            public void run() {
+                try {
+                    Thread.sleep(2000);
+                    Runtime.getRuntime().halt(0);
```

Comment:
This shutdown hook looks a little scary, should have a todo to fix the blocking spot.